### PR TITLE
Assessments: Style student multiple choice table

### PR DESCRIPTION
**Pull request by nkiruka**
This PR is to style the student multiple choice table:

Implementation:

- A purple check mark circle is visible when an assessment is submitted.
- Adjust size of check mark circle
- Set column width for each column
- Set text color for student name to teal

screen shot 2018-05-31 at 12 02 10 pm

### CODE REVIEW

**Comment by maddiedierker** [comment]
it looks like this is a branch off of another branch for a different (still open) PR, so just make sure to merge the other PR first! you can also change the base branch for this one so you only see the diff for this PR rather than the diff for both. if you do change the base branch, just make sure it's set back to `staging` before you merge 😄

**Comment by caleybrock** [comment]
Looks like you've merged staging into this branch, which means you need to change your base back to staging (otherwise you'll have thousands of lines changed) :)

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -1012,6 +1012,7 @@
   "sharingAgePrompt": "Please select an age.",
   "sharingBlockedByTeacher": "Sorry, you do not have permissions to share this project. If you want to be able to share your project, please ask your teacher to enable sharing of App Lab / Game Lab / Web Lab projects for your section from the 'Manage students' tab in their dashboard. They can do this by adding the project sharing column from the Actions settings menu.",
   "show": "Show",
+  "showStudentName": "Student name",
```
Comment (caleybrock) [inline apps/i18n/common/en_us.json]: What's the reason for the "show" part? We would typically just say `studentName` if this text is just a label and not an action.

File: apps/src/templates/sectionAssessments/StudentMultipleChoiceOverviewTable.jsx [hunk 2]
```diff
@@ -5,6 +5,41 @@ import i18n from '@cdo/locale';
...
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import color from "@cdo/apps/util/color";
+
+const TABLE_WIDTH = tableLayoutStyles.table.width;
+const TABLE_COLUMN_WIDTHS = {
+  name: TABLE_WIDTH / 3,
+  numMultipleChoiceCorrect: TABLE_WIDTH / 8,
+  numMultipleChoice: TABLE_WIDTH / 8,
+  percentCorrect: TABLE_WIDTH / 8,
+  timeStamp: TABLE_WIDTH / 5
```
Comment (caleybrock) [inline apps/src/templates/sectionAssessments/StudentMultipleChoiceOverviewTable.jsx]: all the names here match the property name except for `timeStamp`, can we make this one match what the property says too?

File: apps/i18n/common/en_us.json (hunk 1 above)
Comment (nkiruka) [reply apps/i18n/common/en_us.json]: Fixed.  Changed showStudentName to studentNameHeader
Thanks.

File: apps/i18n/common/en_us.json [hunk 3]
```diff
@@ -1066,6 +1066,7 @@
   "studentControl": "Individual student control",
   "studentLockStateInstructions": "Use the table below to control the state for each individual student.",
   "studentName": "Student Name",
+  "studentNameHeader": "Student name",
```
Comment (caleybrock) [inline apps/i18n/common/en_us.json:1069]: 👍

**Pull request merged**